app/components/Statistics/index.tsx

```diff
@@ -0,0 +1,314 @@
+import moment from 'moment-timezone';
+import qs from 'qs';
+import { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import Select from 'react-select';
+import {
+  Bar,
+  BarChart,
+  CartesianGrid,
+  Legend,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import PieChartWithLabel from 'app/components/Chart/PieChartWithLabel';
+import type { DistributionDataPoint } from 'app/components/Chart/utils';
+import { CHART_COLORS } from 'app/components/Chart/utils';
+import { Content } from 'app/components/Content';
+import { selectStyles, selectTheme } from 'app/components/Form/SelectInput';
+import { Flex } from 'app/components/Layout';
+import LoadingIndicator from 'app/components/LoadingIndicator';
+import type { Dateish, Event, Group } from 'app/models';
+import { EVENT_CONSTANTS } from 'app/routes/events/utils';
+import styles from './Statistics.css';
+
+interface Props {
+  committees: Group[];
+  revueGroups: Group[];
+  previousEvents: Event[];
+  onSelect: (year: string, semester: string) => void;
+}
+
+interface WeekNumberDistribution {
+  weekNumber: string;
+}
+
+interface EventTypeDistribution {
+  eventType: string;
+  count: number;
+  paidCount: number;
+  capacity: number;
+  registrationCount: number;
+}
+
+interface StatisticsDistributions {
+  eventTypeDistribution: EventTypeDistribution[];
+  weekNumberDistribution: WeekNumberDistribution[];
+}
+
+interface EventTypeDistributions {
+  capacityDistribution: DistributionDataPoint[];
+  paidDistribution: DistributionDataPoint[];
+}
+
+const SEMESTER_OPTIONS = [
+  { label: 'hele året', value: 'year' },
+  { label: 'vår', value: 'spring' },
+  { label: 'høst', value: 'autumn' },
+];
+
+const EVENT_TYPE_OPTIONS = Object.keys(EVENT_CONSTANTS).map((type) => ({
+  label: EVENT_CONSTANTS[type],
+  value: type,
+}));
+
+const PAST_FIVE_YEARS_OPTIONS = [0, 1, 2, 3, 4].map((yearOffset) => {
+  const year = moment().year() - yearOffset;
+  return { value: year, label: String(year) };
+});
```

Comment:
Doesn't really matter, but

```suggestion
const PAST_FIVE_YEARS_OPTIONS = [...Array(5)].map((_, yearOffset) => {
  const year = moment().year() - yearOffset;
  return { value: year, label: String(year) };
});
```

Reply:
sure, fancyness for the win